controllers/openshift_controller.go

```diff
@@ -936,13 +936,17 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigInstallRequest() (ctrl.
 	if mcfgv1.IsMachineConfigPoolConditionTrue(foundMcp.Status.Conditions, mcfgv1.MachineConfigPoolUpdated) &&
 		foundMcp.Status.ObservedGeneration > r.kataConfig.Status.BaseMcpGeneration &&
 		foundMcp.Status.UpdatedMachineCount == foundMcp.Status.MachineCount {
-		r.Log.Info("set runtime class")
+		r.Log.Info("create runtime class")
 		r.kataConfig.Status.InstallationStatus.IsInProgress = "false"
-		return r.setRuntimeClass()
+		err := r.createRuntimeClass()
+		if err != nil {
+			return reconcile.Result{Requeue: true, RequeueAfter: 15 * time.Second}, err
+		}
```

Comment:
@gkurz you are right. I put the runtimeclass changes as separate commits. Since I had to change the signature, I thought of making changes to the method name as well. I can create a different PR for the runtimeclass method changes.  Guess this will be preferred as well